builder/lxc/step_export.go

```diff
@@ -108,6 +133,31 @@ func (s *stepExport) Run(state multistep.StateBag) multistep.StepAction {
 	return multistep.ActionContinue
 }
 
+func (s *stepExport) PrepareExport(containerDir string, exportFolders []ExportConfig) (error, string) {
+	containerDir = filepath.Join(containerDir, "rootfs")
+	exportFolder := filepath.Join(containerDir, "lxc-export-container-dir")
+	err := s.SudoCommand([]string{ "mkdir", "-p", exportFolder}...)
+	if err != nil {
+		return nil, exportFolder
+	}
+	for i := 0; i < len(exportFolders); i++ {
+		src := filepath.Join(containerDir, exportFolders[i].Src)
+		dest := filepath.Join(exportFolder, exportFolders[i].Dest)
+		destFolder := filepath.Dir(dest)
+		if destFolder != exportFolder {
+			err := s.SudoCommand([]string{ "mkdir", "-p", destFolder}...)
+			if err != nil {
+				return err, exportFolder
+			}
+		}
+		err := s.SudoCommand([]string{ "mv", src, dest}...)
+		if err != nil {
+			return err, exportFolder
+		}
+	}
+	return nil, exportFolder
+}
+
 func (s *stepExport) Cleanup(state multistep.StateBag) {}
```

Comment:
I don't think we need to do any extra steps, since we move things inside the rootfs and when we restart the job we completely remove output-lxc folder.